server/package.json

```diff
@@ -15,6 +15,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "@mueller-splash-pad/knex": "file:../knex",
```

Comment:
is it ok to import a package this way? I didn't want to publish the private package. I'm not a npm expert :(

Reply:
yes! totally fine. It's the new way of importing local packages.